tests/data/fixtures/db.yaml

```diff
@@ -711,7 +759,8 @@
     year: 2020
     version: draft
     cas: 58-08-2
-    assessment_objective: '<p>Example the health effects of Chemical C. Drafted by scientists and contractors at US EPA.</p>'
```

Comment:
remove things between line 714 and 1090. It's not a big deal by themselves, but often times we'll get merge conflicts with other files when there are little things sprinkled in like this.